src/genie/libs/parser/iosxe/tests/ShowLispInstanceIdEthernetServer/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,95 @@
+expected_output = {
+    "instance_id": {
+        4100: {"lisp": 0},
+        8188: {
+            "eid_prefix": {
```

Comment:
Please `site_name` as key.
```suggestion
        8188: {
            "site_name": {
                "site_uci": {
            		"eid_prefix": {
```
because below pattern is possible.
```
site_uci       never     no     --                   8188     any-mac
               2w1d      yes#   10.8.130.4:61275     8188     1416.9d28.c100/48
site_uci2      2w1d      yes#   10.8.130.4:61275     8188     1416.9d28.c2c0/48
               2w1d      yes#   10.8.130.4:61275     8188     1416.9d28.c300/48
site_uci3      2w1d      yes#   10.8.130.4:61275     8188     1416.9d28.c3a0/48
               2w1d      yes#   10.8.130.4:61275     8188     1416.9d28.ea00/48
```

Reply:
Is `site_uci       never     no     --                   8188     any-mac` data we need to capture as well?

Reply:
yes. please capture it as well.

Reply:
Okay that actually makes it easier!

Reply:
Updated!